step/src/main/java/com/google/sps/Message.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.io.*; 
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+/** Variables of a message */
+public final class Message {
+  private final String sender; 
+  private final String recipient; 
+  private final String text; 
+  private final long timestamp;
+
+  public Message(String sender, String recipient, String text, long timestamp) {
+      this.sender = sender;
+      this.recipient = recipient;
```

Comment:
Can you change the sender and recipient variable names to senderId and recipientId? With it being a string, it could get confused as being a name rather than an id.